packages/general/src/util/Error.ts

```diff
@@ -12,6 +12,13 @@ function considerAsError(error: unknown): error is Error {
     return (error as Error).message !== undefined;
 }
 
+export function asError(e: any, prefixText?: string): Error {
+    if (e instanceof Error) {
```

Comment:
It occurs to me that if an error is thrown from another vm context, `instanceof` won't work... Also this doesn't really respect duck typing...

So probably we should accept any value with a `message` property that is a string and cast to Error.

Otherwise toString() might convert useful error information to "[object Object]"

Mainly just mention because this is so low level and fits to do it right

Reply:
adjusted